tests/test_io.py

```diff
@@ -0,0 +1,217 @@
+import configparser
+import logging
+import os
+import shutil
+
+import pytest
+
+import py
+from bidict import bidict
+from edges_io import io
+
+LOAD_ALIASES = bidict(
+    {
+        "ambient": "Ambient",
+        "hot_load": "HotLoad",
+        "open": "LongCableOpen",
+        "short": "LongCableShorted",
+    }
+)
+
+LOGGING = logging.getLogger("edges-io")
+
+
+@pytest.fixture("module")
+def create_testdir(tmp_path_factory):
+    testDir = create_test_env(tmp_path_factory)
+    return testDir
+
+
+@pytest.fixture("module")
+def create_test_env(tmp_path_factory):
+    # Create an ideal observation file using tmp_path_factory
+    pthList = ["Spectra", "Resistance", "S11"]
```

Comment:
This is not a big problem, but it's considered more "pythonic" to have variable names in `snake_case`, and class names in `CamelCase`. Might be good to get used to it :-)